### ISSUE TITLE
map attribute

### ISSUE THREAD

**Issue by anibalcatheringer** [uniVocity/univocity-parsers#287]

**Comment by jbax** [comment]
All attributes in your class must have a @Parsed annotation. You can
provide multiple columns names with it.

You can also define a class-level @Headers, or still a call
parserSettings.setHeaders() with custom header names that match with the
attributes of your class.

Is there anything specific you are having trouble with?

**Comment by anibalcatheringer** [comment]
Yes I'm aware of the annotations. But I want to read the CSV column names from a config file using the Spring property @Value. So I really have to map column to attribute via code..
The file would look something like this:
Pojo.id = "User ID"
Pojo.firstName = "FirstName"

**Comment by jbax** [comment]
Got it. Right now there's no such feature. I'll make this work on the next
release (2.8.0) before we move to java 8.

Thanks for the suggestion!

**Comment by anibalcatheringer** [comment]
Perfect! Thank you!

**Comment by jbax** [comment]
Version 2.8.0 released. Thank you!
